container-runtime/src/Adapters/Systems/SystemForEmbassy/index.ts

```diff
@@ -335,6 +335,113 @@ export class SystemForEmbassy implements System {
       await this.migration(effects, previousVersion, timeoutMs)
     await effects.setMainStatus({ status: "stopped" })
     await this.exportActions(effects)
+    await this.exportNetwork(effects)
```

Comment:
should this not be part of setupInterfaces in MainLoop? or is that in the wrong place?

Reply:
Needs to be ran before... config. So therefore can't be main. Good idea, but then the package known as Vaultwarden shoots you